packages/extension/src/ui/features/accounts/multisig/NewMultisigScreen.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  BarCloseButton,
+  H6,
+  NavigationContainer,
+  P4,
+  icons,
+  logos,
+} from "@argent/ui"
+import { Center, Flex } from "@chakra-ui/react"
+import { FC, useCallback } from "react"
+import { useNavigate } from "react-router-dom"
+
+import { CustomButtonCell } from "../../../components/CustomButtonCell"
+import { routes } from "../../../routes"
+import { assertNever } from "../../../services/assertNever"
+import { useAddAccount } from "../useAddAccount"
+
+const { AddIcon, MultisigJoinIcon } = icons
+const { MultisigDiagram } = logos
+
+type MultisigOptionType = "create" | "join"
+
+interface MultisigOption {
+  title: string
+  subtitle?: string
+  type: MultisigOptionType
+  icon: React.ReactNode
+}
+
+const multisigOptions: MultisigOption[] = [
+  {
+    title: "Create new multisig",
+    type: "create",
+    icon: <AddIcon />,
+  },
+  {
+    title: "Join existing multisig",
+    subtitle: "Create a new signer key",
+    type: "join",
+    icon: <MultisigJoinIcon />,
+  },
+]
+
+export const NewMultisigScreen: FC = () => {
+  const navigate = useNavigate()
+  const { addAccount } = useAddAccount()
+
+  const onClick = useCallback(
+    async (type: MultisigOptionType) => {
+      switch (type) {
+        case "create": {
+          const url = `index.html?goto=${routes.multisigCreate()}`
```

Comment:
Let's keep this simpler and consistent with the ledger version

```suggestion
          const url = `index.html?goto=multisig`
```